contrib_bots/lib/DefineBot.py

```diff
@@ -0,0 +1,79 @@
+# See readme.md for instructions on running this code.
+import json
+import requests
+import logging
+
+class DefineHandler(object):
+    '''
+    This plugin define a word that the user inputs. It
+    looks for messages starting with '@define'.
+    '''
+
+    def __init__(self):
+        self._DEFINITION_API_URL = 'https://owlbot.info/api/v1/dictionary/{}?format=json'
+        self._REQUEST_ERROR_MESSAGE = 'Definition not available.'
+        self._EMPTY_WORD_REQUEST_ERROR_MESSAGE = 'Please enter a word to define.'
+
+    def usage(self):
+        return '''
+            This plugin will allow users to define a word. Users should preface
+            messages with "@define".
+            '''
+
+    def triage_message(self, message):
+        # return True iff we want to (possibly) response to this message
+
+        original_content = message['content']
+
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting follow
+        # ups for own follow ups!
+        is_define = original_content.startswith('@define')
+
+        return is_define
+
+    def _handle_definition(self, original_content):
+        # Remove '@define' from the message and extract the rest of the message, the
+        # word to define.
+        split_content = original_content.split(' ')
+        to_define = ' '.join(split_content[1:])
```

Comment:
Instead of splitting and joining again you could give `maxsplit` argument to `split`:
https://docs.python.org/3/library/stdtypes.html#str.split